tests/conftest.py

```diff
@@ -3,15 +3,29 @@
 import sys
 
 import pytest
+from lxml import etree
 from lxml.builder import ElementMaker
 
-from space_packet_parser.xtce import XTCE_URI, XTCE_NSMAP
+from space_packet_parser import common
+from space_packet_parser.xtce import DEFAULT_XTCE_NSMAP, DEFAULT_XTCE_NS_PREFIX, XTCE_1_2_XMLNS
 
 
 @pytest.fixture(scope="session")
 def elmaker():
     """ElementMaker for testing XML element creation"""
-    return ElementMaker(namespace=XTCE_URI, nsmap=XTCE_NSMAP)
+    return ElementMaker(namespace=XTCE_1_2_XMLNS, nsmap=DEFAULT_XTCE_NSMAP)
+
+
+@pytest.fixture
+def xtce_parser():
+    """Parser for testing that knows about the standard testing namespace we use"""
+    el = common.NamespaceAwareElement
+    el.set_nsmap(DEFAULT_XTCE_NSMAP)
+    el.set_ns_prefix(DEFAULT_XTCE_NS_PREFIX)
+    xtce_element_lookup = etree.ElementDefaultClassLookup(element=el)
+    xtce_parser = etree.XMLParser()
+    xtce_parser.set_element_class_lookup(xtce_element_lookup)
+    return xtce_parser
```

Comment:
@greglucas These are the fixtures that assume the `XTCE_1_2_XMLNS` namespace URI and the default namespace mapping.